tests/test_data.py

```diff
@@ -162,3 +146,20 @@
     "triggersIds": [],
     "alertId": 304255
 }
+
+alert_D = {
+    "id": testId,
+    "createdById": 101997,
+    "createdDate": 1459857033871,
+    "modifiedById": 101997,
+    "modifiedDate": 1459857033871,
+    "name": alertName,
+    "expression": "-1d:hdara:test:sum",
+    "cronEntry": "*/15 * * * *",
+    "enabled": False,
+    "missingDataNotificationEnabled": False,
+    "notificationsIds": [],
+    "triggersIds": [],
+    "ownerName": userName
+}
```

Comment:
You may as well just move this block back to the original location so that the diff is clean and shows the exact changes done.

Reply:
sure, done